### ISSUE TITLE
CNN chapter

### ISSUE THREAD

**Issue by bpesquet** [bpesquet/ainotes#3]
- [ ] Transfer learning example
- [ ] Overfitting assessment using a validation set
- [ ] Conv filters viz
- [ ] Bias in conv filters
